serverless/functions/serverless-developing-go-functions.adoc

```diff
@@ -0,0 +1,27 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-developing-go-functions"]
+= Developing Golang functions
+:context: serverless-developing-go-functions
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+:FeatureName: {FunctionsProductName}
+include::modules/technology-preview.adoc[leveloffset=+2]
+
+After you have xref:../../serverless/functions/serverless-functions-getting-started.adoc#serverless-functions-getting-started[created a Golang function project], you can modify the template files provided to add business logic to your function.
+
+[id="prerequisites_serverless-developing-go-functions"]
+== Prerequisites
+
+* Before you can develop functions, you must complete the set up steps in xref:../../serverless/functions/serverless-functions-setup.adoc#serverless-functions-setup[Setting up {FunctionsProductName}].
```

Comment:
I was trying to decide if "set up steps" should be "set-up steps". But maybe you don't really need it at all and could just leave it as "steps"?